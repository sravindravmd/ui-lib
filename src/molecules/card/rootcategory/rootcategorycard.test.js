import React from 'react'
import { render, cleanup, fireEvent } from '@testing-library/react'
import RootCategoryCard from './index'

afterEach(cleanup)

describe('<RootCategoryCard/>', () => {
  it('should render RootCategoryCard correctly ', () => {
    const { container } = render(
      <RootCategoryCard title='Shirts' subtitle='Updated 9/8 12:15PM PST' />
    )
    expect(container).toMatchSnapshot()
  })

  it('should render RootCategoryCard after click ', () => {
    const { container, getByText } = render(
      <RootCategoryCard
        title='Shirts'
        subtitle='Updated 9/8 12:15PM PST'
        onClick={() => {}}
      />
    )
    fireEvent.click(getByText('Shirts'))
    expect(container).toMatchSnapshot()
  })
})
