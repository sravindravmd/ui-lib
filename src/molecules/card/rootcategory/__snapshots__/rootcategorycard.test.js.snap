// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RootCategoryCard/> should render RootCategoryCard after click  1`] = `
<div>
  <div
    class="sc-bdfBwQ bFqsTc"
  >
    <div
      class="sc-eCssSg cqgeUx"
      tabindex="-1"
    >
      <div
        class="sc-gsTCUz dbgDwb"
      >
        <div
          class="sc-dlfnbm GbhKX"
        >
          Shirts
        </div>
        <div
          class="sc-hKgILt cUJNqp"
        >
          Updated 9/8 12:15PM PST
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`<RootCategoryCard/> should render RootCategoryCard correctly  1`] = `
<div>
  <div
    class="sc-bdfBwQ rKQoS"
  >
    <div
      class="sc-eCssSg cqgeUx"
      tabindex="-1"
    >
      <div
        class="sc-gsTCUz dbgDwb"
      >
        <div
          class="sc-dlfnbm GbhKX"
        >
          Shirts
        </div>
        <div
          class="sc-hKgILt cUJNqp"
        >
          Updated 9/8 12:15PM PST
        </div>
      </div>
    </div>
  </div>
</div>
`;
